app/helpers/application_helper.rb

```diff
@@ -1,9 +1,49 @@
+require 'redcarpet'
+require 'redcarpet/render_strip'
+
+# Very barebone "markdown" renderer that ignores all elements
+# except paragraphs
+class MarkdownRenderTruncate < Redcarpet::Render::Base
+  def paragraph(text)
+    text + ' '
+  end
+
+  def link(link, title, content)
+    content
+  end
+end
+
 module ApplicationHelper
   def menu_items
     (menu_item t(:events, scope: 'navbar'), events_path) +
     (menu_item t(:requests, scope: 'navbar'), requests_path)
   end
 
+  # Render the given string as markdown
+  #
+  # @param text String containing markdown
+  # @param truncatable boolean leave only paragraph elements
+  # @return string html_safe rendered markdown
+  def markdown(text, truncatable = false)
```

Comment:
Well it doesn't truncate anything, it just creates a string suitable for truncation :P I intentionally left out the actual truncation because rails' `truncate` function has various options on how the truncation should happen and I didn't want to limit those (e.g. where the "read on" link is supposed to point to).